### ISSUE TITLE
Добавить рендер ссылок

### ISSUE THREAD

**Issue by OlegShchavelev** [OlegShchavelev/ALTRegularGnomeWiki#46]
Немножко улучшим отображение ссылок, компонент:
https://nolebase-integrations.ayaka.io/pages/en/integrations/markdown-it-bi-directional-links/

**Comment by Ampernic** [comment]
Так же требуется решить вопрос со структурой и путями

**Comment by Ampernic** [comment]
Так, немного задокументирую что было по этому ишью за все время.

> Требуется решить вопрос со структурой и путями

Не требуется, так как первоначально пошел по ложному пути.
Рендер ссылок это не `bi-directional-links`, а `inline-link-preview`.
Со всторым все получилось, но компонент работает малость не так как нужно, а именно не может отсылаться на определенный фрагмент статьи по тегу раздела.

На данный момент есть ишью авторам https://github.com/nolebase/integrations/issues/157
Стоит в целях на релиз версии `2.1.0`

Ранее обсуждалось в чате AGW: 6 Апреля 2024

![Image](https://github.com/OlegShchavelev/ALTGnomeWiki/assets/44705058/3ce1427c-c3b9-49a1-9d15-210a4efc1f24)
![Image](https://github.com/OlegShchavelev/ALTGnomeWiki/assets/44705058/61b2c3f7-f196-4f1a-b31e-8cec09de61c8)
![Image](https://github.com/OlegShchavelev/ALTGnomeWiki/assets/44705058/0bbc3fad-4eea-4802-97a6-3027fc958096)
